.github/workflows/cypress-test-oidc-e2e.yml

```diff
@@ -60,7 +60,6 @@ jobs:
           else
               echo "Checksum match confirmed. Proceeding with setup."
           fi
-          chmod +x ./kc.sh
```

Comment:
This is a duplicate line. Its also done up above. I am removing the duplicate.

Reply:
Actually, can we remove the above line L51 instead of this one? Cuz I think the intention was to double check the checksum and proceed to make the script executable if the checksum matched.